docs/components/zeebe/technical-concepts/clustering.md

```diff
@@ -42,6 +42,8 @@ This may lead to one node becoming the leader for all partitions. This is not a
 To reach a well-distributed leadership again, the [Rebalancing API](../../../self-managed/zeebe-deployment/operations/rebalancing.md) can be used. Be aware that this is on a best-effort basis.
 :::
 
+<!--- Is this Rebalancing API available for Camunda 8 SaaS environments, or only Self-Managed? --->
+
```

Comment:
It is not available for SaaS users. But it is available internally, and we use it sometimes in support/incidents.